doc/adr/0001-reactive-reassignment.md

```diff
@@ -0,0 +1,49 @@
+# 1. Reactive reassignment
+
+Date: 2018-11-21
+
+## Status
+
+Accepted
+
+## Context
+
+In current architecture, we receive always a batch of products. For this batch, we check if any products need reassignment.
+In order to find out, we search for conflicting products with query by SKUs from variants and slugs from all locale and both current and staged.
+This query can get too big for GET request and cannot be used.  
```

Comment:
I would change `and cannot be used` to something like `which can fail due to max query length limit`

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/77/commits/6b966f6aee1249d5398b4b636b6c9803f78e56ef#diff-69de50b194fdf22d40e38b82f2045440R13